KeenSwiftClientExample/KeenSwiftClientExample/AppDelegate.swift

```diff
@@ -8,19 +8,30 @@
 
 import UIKit
 
+class ExampleLogger: KeenLogSink {
```

Comment:
Should we go as far as shipping some examples that do logging via 3rd-party loggers/reporting solutions (like crashlytics), or is that easy enough or sufficiently customized that the exercise should be left to the reader?